libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/NavigationCamera.java

```diff
@@ -233,20 +233,51 @@ public void showRouteOverview(@NonNull int[] padding) {
    */
   public boolean showRouteGeometryOverview(@NonNull int[] padding) {
     updateCameraTrackingMode(NAVIGATION_TRACKING_MODE_NONE);
-    DirectionsRoute currentRoute;
-    if (currentRouteProgress != null) {
-      currentRoute = currentRouteProgress.getRoute();
-    } else if (currentRouteInformation != null) {
-      currentRoute = currentRouteInformation.getRoute();
+    final DirectionsRoute currentRoute = getRoute();
+    if (currentRoute == null) {
+      Timber.e("Unable to show route overview, the route is null.");
```

Comment:
Could you extract this magic number (duplicated in `showRouteGeometryRemainingOverview`) into a constant?

Reply:
constant added.